baker/SiteBaker.tsx

```diff
@@ -633,6 +655,24 @@ export class SiteBaker {
         this.progressBar.tick({ name: "✅ baked google doc posts" })
     }
 
+    async bakeGDocTombstones(knex: db.KnexReadonlyTransaction) {
+        if (!this.bakeSteps.has("gdocTombstones")) return
+        const tombstones = await getTombstones(knex)
+
+        for (const tombstone of tombstones) {
+            const attachments = await this.getPrefetchedGdocAttachments(knex)
+            try {
+                await this.bakeOwidGdocTombstone(tombstone, attachments)
```

Comment:
It's possible to set an invalid `relatedLink` (with either an unpublished or unregistered gdoc) in the admin

We should throw an error during the save attempt if the `relatedLink` is invalid, and then also log an error to Bugsnag here if the `relatedLink` is invalid during bake (i.e. the `relatedLink` gdoc has been unpublished)

`ProminentLink` has a graceful fallback so I think just making sure people know it's happening through Bugsnag is sufficient.